examples/bal-to-edi/records-to-edi.bal

```diff
@@ -0,0 +1,12 @@
+import ballerina/io;
+import edi_to_bal.sorder;
+
+public function main() returns error? {
+    sorder:SimpleOrder simpleOrder = 
+        {header: {code: "HDR", orderId: "ORDER_200", organization: "HMart", date: "17-05-2023"}};
+    simpleOrder.items.push({code: "ITM", item: "A680", quantity: 15}); 
+    simpleOrder.items.push({code: "ITM", item: "A530", quantity: 2}); 
+    simpleOrder.items.push({code: "ITM",item: "A500", quantity: 4});
```

Comment:
Can't these be specified when creating `simpleOrder` itself?

Reply:
Yes, it is possible. But then it would look like hard-coding a json. I used the push method to show the flexibility of working with records and converting those to EDI.